cmd/oc-mirror/main.go

```diff
@@ -2,9 +2,10 @@ package main
 
 import (
 	"flag"
-	"github.com/RedHatGov/bundle/pkg/cli/mirror"
-	"github.com/sirupsen/logrus"
 	"io/ioutil"
+
+	"github.com/openshift/oc-mirror/pkg/cli/mirror"
```

Comment:
Personal nit. I like to organize same-module packages in the lowest import section. Feel free to ignore.
```go
import (
	"stdpkg"

	"github.com/external/module"

	"github.com/openshift/oc-mirror/pkg/cli/mirror"
)
```